assets/lib/share-link/share-link.js

```diff
@@ -144,9 +132,15 @@
 		return link;
 	};
 
-	$.fn.shareLink = function( settings ) {
-		return this.each( function() {
-			$( this ).data( 'shareLink', new ShareLink( this, settings ) );
+	Element.prototype.shareLink = function ( settings ) {
+		new ShareLink( this, settings );
+		return this;
+	};
+
+	NodeList.prototype.shareLink = function ( settings ) {
+		this.forEach( ( element ) => {
+			new ShareLink( element, settings );
 		} );
+		return this;
 	};
-} )( jQuery );
+} )();
```

Comment:
Maybe?

```suggestion
} );
```

Reply:
@hein-obox but don't we need to call the wrapping function? if we remove the `()` at the end, will the function get called ?

Reply:
@nuritsha Could you check how we do it other places? Generally speaking we export the class or function, and we don't have to wrap it in inside the kind of jQuery function.